inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/param_result_custom_blob.cpp

```diff
@@ -21,7 +21,7 @@ class ParameterResultCustomBlobTest : public ParameterResultSubgraphTest {
         const size_t elementsCount = inputBlob->size();
         for (size_t i = 0; i < inferIterations; ++i) {
             CommonTestUtils::fill_data_random<Precision::FP32>(inputBlob, 10, 0, 1, i);
-            const auto& inputsInfo = cnnNetwork.getInputsInfo().begin()->second;
+            auto inputsInfo = cnnNetwork.getInputsInfo().begin()->second;
```

Comment:
Could you please also remove explicit memory allocation and use vector or unique ptr? In case if infer below or other methods throw an exception, we still have memory leak

Reply:
The intention was to clearly show that we use dynamically allocated memory. And if an exception is thrown the test will fail anyway. But, since you are already the second person who noticed this design flaw, and some code analysis tools might be triggered, I decided to rewrite this behavior.